src/olympia/blocklist/tasks.py

```diff
@@ -130,3 +139,20 @@ def import_block_from_blocklist(record):
         kinto_import.outcome = KintoImport.OUTCOME_NOMATCH
         log.debug('Kinto %s: No addon found', kinto_id)
     kinto_import.save()
+
+
+@task
+def upload_mlbf_to_kinto(generation_time):
```

Comment:
I notice there was some logging in `upload_mlbf_to_kinto` in `cron.py`. Should there be any logging in this task as well?

Reply:
There's default celery logging whenever a task is picked up and started, and logging in `KintoServer` too so I didn't add any more (or remove any) when I split these lines out.